Gemfile

```diff
@@ -3,12 +3,4 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in tmuxinator.gemspec
 gemspec
 
-gem "rake", "~> 10.4"
-gem "rspec", "~> 3.3"
-gem "simplecov", "~> 0.11.0"
-gem "coveralls", "~> 0.7"
-gem "awesome_print", "~> 1.2"
-gem "pry", "~> 0.10"
-gem "factory_girl", "~> 4.5"
 gem "rubocop", "0.35.1", require: false
```

Comment:
I somehow missed this in my first pass -- is there a reason RuboCop wasn't moved to the gemspec?